tests/integration/test_cogs_bans.py

```diff
@@ -42,7 +29,10 @@ async def test_bans_webpage_changed(bans_cog, configured_httpserver, last_bans):
             configured_httpserver.url_for("/bansummary-changed.txt"),
             headers={"Pragma": "no-cache", "Cache-Control": "no-cache"},
         ) as resp:
+            assert resp.status == 200
```

Comment:
Funny enough, without this assertion, the test was previously incorrectly passing;

The `bansummary-changed.txt` url didn't actually exist, so the `resp` was coming back with a 500 server error; but there was still some error text returned by `resp.text()`. And because the text didn't match, it was indicating a webpage changed. So basically it was testing nothing and passing anyway.

Reply:
Good catch.